project/user/serializers.py

```diff
@@ -106,3 +108,68 @@ def create(self, validated_data):
 
         user = super().create(validated_data)
         return user
+
+
+class FriendRequestCreateSerializer(serializers.ModelSerializer):
+    sender_profile = serializers.SerializerMethodField()
+
+    class Meta:
+        model = FriendRequest
+        fields = "__all__"
+
+    def create(self, validated_data):
+        sender = validated_data.get("sender")
+        receiver = validated_data.get("receiver")
+        friend_request, is_created = FriendRequest.objects.update_or_create(
+            sender=sender, receiver=receiver, defaults={"created": datetime.now()}
+        )
+        return friend_request
+
+    def validate(self, data):
+        receiver = data.get("receiver")
+        sender = data["sender"]
+        if sender.friends.filter(pk=receiver.id).exists():
+            raise serializers.ValidationError("이미 친구입니다.")
+        if (
+            FriendRequest.objects.all()
+            .filter(sender=receiver, receiver=sender)
+            .exists()
+        ):
+            raise serializers.ValidationError("이 유저에게 친구 요청을 받았습니다.")
```

Comment:
```suggestion
            raise serializers.ValidationError("이 유저에게 이미 친구 요청을 받았습니다.")
```
로 고치면 의미가 더 명확할 것 같습니다!